bzt/utils.py

```diff
@@ -382,27 +388,37 @@ def is_ready(self):
             self.name = self.fds.name
             return True
 
+    def decode(self, line):
```

Comment:
`self.decode(line)` can probably be a protected method. there's no need to expose it, as it's only going to be used by other methods.

Reply:
there is possibility it can be useful for those who call get_bytes (works without decoding) as FileReader can have some insider info about file contents..